### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by gnowxilef**

### CODE REVIEW

**Comment by gnowxilef** [comment]
poke?

File: kochira/services/core/onconnect.py [hunk 1]
```diff
@@ -12,11 +12,11 @@
...
 class Config(Config):
-    commands = config.Field(doc="List of commands to run.",
-                            type=config.Many(list))
+    commands = config.Field(doc="List of commands to run.")
 
 
 @service.hook("connect", priority=-10)
 def onconnect(ctx):
     for command in ctx.config.commands:
-        ctx.client.rawmsg(*command)
```
Comment (Shizmob) [inline kochira/services/core/onconnect.py]: Why this change?

File: kochira/services/core/onconnect.py [hunk 2]
```diff
@@ -12,11 +12,11 @@
 
 @service.config
 class Config(Config):
-    commands = config.Field(doc="List of commands to run.",
-                            type=config.Many(list))
+    commands = config.Field(doc="List of commands to run.")
```
Comment (Shizmob) [inline kochira/services/core/onconnect.py]: Why omit this?

File: requirements.txt [hunk 3]
```diff
@@ -16,3 +16,5 @@ pathlib
 https://github.com/rfk/pyenchant/tarball/master#egg=pyenchant
 textblob
 ccy
+argparse==1.2.1
```
Comment (Shizmob) [inline requirements.txt]: `argparse` should be standard in all feasibly deployed Python 3 distributions, so this seems unnecessary.

File: kochira/services/core/onconnect.py (hunk 1 above)
Comment (gnowxilef) [inline kochira/services/core/onconnect.py]: rawmsg takes a specific command argument: https://pydle.readthedocs.org/en/latest/api/client.html#pydle.BasicClient.rawmsg.

File: requirements.txt (hunk 3 above)
Comment (gnowxilef) [inline requirements.txt]: that is true, but this specific version had to be installed for kochira to work correctly.

File: kochira/services/core/onconnect.py (hunk 2 above)
Comment (gnowxilef) [inline kochira/services/core/onconnect.py]: otherwise the commands list/array is an array of individual characters, instead of an array/list of commands.

File: requirements.txt (hunk 3 above)
Comment (gnowxilef) [inline requirements.txt]: odd, tried installing a fresh copy and didn't need argparse. removed from commit.
